src/main/java/Aaron/Storage/Storage.java

```diff
@@ -0,0 +1,88 @@
+package Aaron.Storage;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import Aaron.Exception.AaronBotException;
+import Aaron.Task.Deadline;
+import Aaron.Task.Event;
+import Aaron.Task.Task;
+import Aaron.Task.TaskList;
+import Aaron.Task.Todo;
+
+/**
+ * This class writes to and loads existing tasklist files for the aaronbot
+ * @param fileToPath
+ * @param taskList
+ * @throws IOException
+ */
+public class Storage {
+    public static void writeToFile(String filePath, TaskList taskListWrite) throws IOException{
+        Path path = Paths.get(filePath);
+        Files.createDirectories(path.getParent());
+        File prevTaskList = new File(filePath);
+        prevTaskList.createNewFile();
+        if (prevTaskList.exists()) {
+            prevTaskList.delete();
+        }
+        try {
+            FileWriter writer = new FileWriter(filePath, true);
+            for (int i = 1; i < taskListWrite.getTasklistSize() + 1; i++) {
+                String taskDesc = taskListWrite.printTask(i).toString();
+                writer.write(taskDesc + System.lineSeparator());
+            }
+            writer.close();
+        } catch (IOException e) {
+            throw new IOException("Error loading file");
+        }
+    }
+
```

Comment:
In the writeToFile method, the use of FileWriter in append mode (new FileWriter(filePath, true)) is highlighted as redundant due to the explicit file deletion logic. Append mode inherently handles the creation of a new file if it doesn't exist, and appends data to the end of an existing file. Therefore, the manual deletion of the file prior to writing is unnecessary and could be removed to streamline the process.